### PR TITLE
Add all marketpay APIs

### PR DESCRIPTION
**Pull request by rikterbeek**
**Description**

**Tested scenarios**

**Fixed issue**:

### CODE REVIEW

File: src/Adyen/Service/ResourceModel/Account/GetUploadedDocuments.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    protected $_endpoint;
+
+    /**
+     * GetUploadedDocuments constructor.
+     * @param $service
+     */
+    public function __construct($service)
+    {
+        $this->_endpoint = $service->getClient()->getConfig()->get('endpointAccount') . '/' . $service->getClient()->getApiAccountVersion() . '/getUploadedDocuments';
+        parent::__construct($service, $this->_endpoint, $this->_requiredFields);
```
Comment (Aleffio) [inline src/Adyen/Service/ResourceModel/Account/GetUploadedDocuments.php]: please remove the requiredfields

Comment (rikterbeek) [reply src/Adyen/Service/ResourceModel/Account/GetUploadedDocuments.php]: yes will fix this

File: tests/MockTest/NotificationTest.php [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+        $params = json_decode('
+            {
+              "notificationIds": [
+                15007,
+                15008
+              ]
+            }',
+            true
+        );
+        $result = $service->updateNotificationConfiguration($params);
```
[the remark this answers is not in the document]
Comment (rikterbeek) [reply tests/MockTest/NotificationTest.php]: thanks will fix this.

File: src/Adyen/Client.php [hunk 3]
```diff
@@ -17,231 +17,247 @@ class Client
...
+    const API_ACCOUNT_VERSION = "v4";
+    const API_FUND_VERSION = "v3";
     const ENDPOINT_TERMINAL_CLOUD_TEST = "https://terminal-api-test.adyen.com";
     const ENDPOINT_TERMINAL_CLOUD_LIVE = "https://terminal-api-live.adyen.com";
     const ENDPOINT_CHECKOUT_TEST = "https://checkout-test.adyen.com/checkout";
     const ENDPOINT_CHECKOUT_LIVE_SUFFIX = "-checkout-live.adyenpayments.com/checkout";
     const ENDPOINT_PROTOCOL = "https://";
+    const ENDPOINT_NOTIFICATION_TEST = "https://cal-test.adyen.com/cal/services/Notification";
+    const ENDPOINT_NOTIFICATION_LIVE = "https://cal-live.adyen.com/cal/services/Notification";
+    const ENPOINT_ACCOUNT_TEST = "https://cal-test.adyen.com/cal/services/Account";
```
[the remark this answers is not in the document]
Comment (rikterbeek) [reply src/Adyen/Client.php]: thanks.

**Comment by rikterbeek** [comment]
@cyattilakiss let's do a seperate PR to fix it across all fields.

**Pull request merged**